CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## v3.21.1
```

Comment:
In PR's we no write version. This line prepends automatically with spatial github action named [publish](https://github.com/ydb-platform/ydb-go-sdk/blob/master/.github/workflows/publish.yml)
Please, remove this line